mvnw/src/test/java/org/eolang/InspectTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang;
+
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+/**
+ * Smoke test to ensure that {@link Inspect} server starts without throwing exceptions.
+ * <p>
+ * This test launches the server in a separate thread to verify startup stability.
+ * Functional behavior is tested in {@code test_inspect.js}.
+ * </p>
+ *
+ * @since 0.29.0
+ */
+final class InspectTest {
+    /**
+     * Verifies that {@link Inspect#main(String[])} starts without errors.
+     *
+     * @throws Exception If the thread is interrupted
+     */
+    @Test
+    void runsWithoutExceptions() throws Exception {
+        final Thread server = new Thread(
+            new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        Inspect.main();
+                    } catch (final IOException ex) {
+                        throw new IllegalStateException("IOException while starting server", ex);
+                    }
+                }
+            }
+        );
+        server.setDaemon(false);
```

Comment:
@ErnestMatskevich Why do we need this? From https://stackoverflow.com/a/17878439/10423604:

> Any thread created by main thread, which runs main method in Java is by default non daemon because Thread inherits its daemon nature from the Thread which creates it i.e. parent Thread and since main thread is a non daemon thread, any other thread created from it will remain non-daemon until explicitly made daemon by calling setDaemon(true).